frontend/src/hooks/useNewsFeed.ts

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from 'react';
+
+interface NewsItem {
+  id: string;
+  fieldData: {
+    'expiry-date': string;
+    'publish-date': string;
+    order: number;
+    'link-text': string;
+    message: string;
+    name: string;
+    'link-url': string;
+    slug: string;
+    segment: string;
+  };
+}
+
+export const useNewsFeed = (segment?: 'redpanda' | 'kafka') => {
```

Comment:
Should we use react query? That way we don't need custom wrapping around fetch()